Packs/Shodan/ReleaseNotes/1_2_3.md

```diff
@@ -3,4 +3,8 @@
 
 ##### Shodan v2
 
-- Metadata and documentation improvements.
+- Improved implementation of the **search** command
```

Comment:
```suggestion
- Improved implementation of the ***search*** command.
```

Reply:
I restored the 1_2_3.md RN from master that had not been merged into my dev branch and bumped my contribution to 1.2.4.  I incorporated your suggested change in that RN.